user_templates/mcu_defaults/ercf/Fysetc_ERCF_ERB.cfg

```diff
@@ -34,4 +34,3 @@ aliases:
     EXTRA_PINS9=MCU_ADC0      , EXTRA_PINS10=MCU_ADC1     ,
     EXTRA_PINS11=MCU_ADC2     , EXTRA_PINS12=MCU_ADC3     ,
     EXTRA_PINS13=<GND>        , EXTRA_PINS14=<5V>         ,
-
```

Comment:
Can you put back this second free new line at the end? It was there on purpose to allow better separation when injecting the templates from the install script